### ISSUE TITLE
404 page padding

### ISSUE THREAD

**Issue by alifeee** [lipu-tenpo/liputenpo.org#64]
![image](https://github.com/lipu-tenpo/liputenpo.org/assets/13833017/42679321-4650-4d17-bed7-1ee25950db11)

**Comment by ngsilverman** [comment]
Note: this only happens if the height of the window is small enough (around 800px).

**Comment by ngsilverman** [comment]
So I fixed the padding, but I should point out I had to go to `/ala` to see the 404 page. Going to arbitrarily erroneous locations, like `/thisdoesntexist`, does not display the 404 page. Just a short message like "Cannot GET /thisdoesntexist". If that's not the desired behavior then I can look into fixing it as well (here's some related documentation https://www.11ty.dev/docs/quicktips/not-found/).
